table-module/src/main/java/com/iluwatar/tablemodule/UserSchemaSql.java

```diff
@@ -0,0 +1,26 @@
+package com.iluwatar.tablemodule;
+
+/**
+ * User Schema SQL Class.
+ */
+public final class UserSchemaSql {
```

Comment:
Having a separate class for a couple of constants feels unnecessary. Could we combine this to `UserTableModule`?